src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -36,12 +37,14 @@ const RequestTable = ({
 
   const loadedRegions = useAppSelector(getLoadedRegions);
 
+  const [showCopied, setShowCopied] = useState(false);
```

Comment:
Why is this state here? Doesn't look like it's being read or written.

Reply:
Ah ! I missed removing the old changes. Thanks for spotting